src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/MainResources_zh_CN.properties

```diff
@@ -72,6 +72,7 @@ error.tool-old-version=\u627E\u4E0D\u5230 {0} {1}\u6216\u66F4\u65B0\u7248\u672C
 error.tool-old-version.advice=\u8BF7\u5B89\u88C5 {0} {1}\u6216\u66F4\u65B0\u7248\u672C
 error.jlink.failed=jlink \u5931\u8D25\uFF0C\u51FA\u73B0 {0}
 error.blocked.option=\u4E0D\u5141\u8BB8\u5728 --jlink-options \u4E2D\u4F7F\u7528 jlink \u9009\u9879 [{0}]
+error.no.name=Name not specified with --name and cannot infer one from app-image
```

Comment:
Do we need to submit a follow-up bug for localization team that we have new non-localized messages in the region specific properties file? I see that foreign app image warning is still not localized.

Reply:
Localization team will translate all changed messages together without individual follow-up bugs, as long as changed before the "localization freeze"